src/assets/css/main.css

```diff
@@ -43,109 +43,147 @@ body{
     user-select: none;
 }
 
-#buttons {
-    display: grid;
-    grid-template-columns: 1fr 1fr 1fr 1fr;
-}
+#calculator-buttons{
+    width: auto;
+    height: auto;
+    border-radius: 0 0 8px 8px;
 
-#buttons button {
-    padding: 20px 0px 20px 0px;
+}
+button{
+    width: 112px;
+    height: 80px;
     background-color: #fafafa;
     border: none;
     font-size: 1.5rem;
     cursor: pointer;
     text-align: center;
     border-radius: 4px;
     user-select: none;
-    margin: 4px;
+    margin: 6px 2px 6px 4px;
 }
-#buttons .left{
+
+#buttons-top,
```

Comment:
do this for buttons inside of calculator, apply this for the rest of your file

Reply:
it's done